contributing.md

```diff
@@ -1,9 +1,126 @@
-# Contributing to the p5.js web editor
+# Contributing to the p5.js web editor 
 
-We welcome community contributions to the p5.js web editor! You can contribute even if you are not a developer by documenting bugs, proposing new features, and updating documentation.
+Hi! Thanks for wanting to contribute to P5's web editor! You are awesome! 😊
 
-When opening an issue, be sure to clearly describe the issue, task, or feature. If it is a bug, try to include steps to reproduce or other contextual information, such as the browser or operating system. 
+This **web editor** for **p5.js**, a JavaScript library with the goal of making coding accessible to **artists**, **designers**, **educators**, and **beginners**, is used to help develop using p5.js library in web browser and thus makes for a good starting point for working with p5. 
```

Comment:
**p5.js** is a JavaScript library with the goal of making coding accessible to **artists**, **designers**, **educators**, and **beginners**. The **p5.js web editor** is an environment to make p5.js sketches without needing to download any software or do any configuration, which makes it a great place to start learning how to code and start learning p5.js.